src/web/hoc/Color/index.js

```diff
@@ -3,8 +3,8 @@ import PropTypes from 'prop-types';
 
 export default function colorify(Component) {
   function Color(props) {
-    const classes = props.className + props.color && ` is-${props.color}`;
-    return <Component {...props} className={classes} />;
+    const classes = props.className + (props.color && ` is-${props.color}`);
+    return <Component {...props} className={classes.trim()} />;
```

Comment:
`trim()`here is a trick ?

Reply:
The goal is to eliminate the gap between className and first class when className is empty.
But well if it's considered a trick :sob:  I can find another way :smile: